app/views/spree/admin/shared/_product_sub_menu.html.haml

```diff
@@ -0,0 +1,9 @@
+- content_for :sub_menu do
+  %ul#sub_nav.inline-menu{"data-hook" => "admin_product_sub_tabs"}
+    = tab :products, :match_path => '/products'
+    = tab :option_types, :match_path => '/option_types'
+    = tab :properties
+    = tab :prototypes
+    = tab :variant_overrides, url: main_app.admin_inventory_path, match_path: '/inventory'
+    -# Commenting out for now, until product import is finished
+    -# = tab :product_import, label: "Import", url: main_app.admin_product_import_path, match_path: '/product_import'
```

Comment:
Hey @luisramos0 @sauloperez this was introduced in `add_product_import_tab.html.haml.deface` but commented out. Should we keep it as is or remove it ?

Reply:
oh no, sorry @HugsDaniel I just realized that @Matt-Yorkley  has already done this as part of https://github.com/openfoodfoundation/openfoodnetwork/pull/2947/files

I hope I didnt waste to much of your time!!!
#2947 will be merged soon and we should build on top of it.

Reply:
I quit

Reply:
😭